lib/elb/ELB.rb

```diff
@@ -69,6 +101,48 @@ def init_elbs
 
         Hash[elbs.flatten.map { |elb| [elb.load_balancer_name, elb] }]
       end
+
+      # Internal: Load ELB tags and map them to their names
+      #
+      # Returns the ELB tags mapped to ELB name
+      def init_elb_tags
+        tags = []
+        elbs.keys.each_slice(20) do |names|
```

Comment:
Do we always want to do all the tags for all the elbs? Or would it be better to do elbs one at a time?